package.json

```diff
@@ -57,9 +57,12 @@
     "handlebars": "^4.4.5",
     "https-proxy-agent": "^2.2.3",
     "3box/ipfs/ipld-zcash/zcash-bitcore-lib/lodash": "^4.17.12",
+    "marked": "^0.7.0",
```

Comment:
These were added to `"resolutions"` as they are deep sub-deps of other packages.